packages/marketplace/README.md

```diff
@@ -183,6 +183,49 @@ everything inside it and manage the entire deploy strategy from one place.
 2. You must add end-to-end tests ahead of deploying your package. Importantly,
    these tests should verify deployment and initialization configuration
 
+# Release
+
+We use release-it for versioning and package publishing.
+
+To install it, run:
+
+```shell
+yarn add -D @release-it/keep-a-changelog release-it
+```
+
+The configuration is done in the `package.json` file.
+
+Here is how it should be set up:
+
+```json
+ "release-it": {
+        "git": {
+            "commitMessage": "chore: @sandbox-smart-contracts/marketplace release v${version}",
+            "tagAnnotation": "@sandbox-smart-contracts/marketplace release v${version}",
+            "tagName": "@sandbox-smart-contracts/marketplace@v${version}"
+        },
+        "plugins": {
+            "@release-it/keep-a-changelog": {}
+        },
+        "hooks": {
+            "before:init": [
+                "yarn lint",
+                "yarn test"
+            ]
+        }
+    },
+```
+
+Steps to create a new release:
+
+1. Add the changes to the CHANGELOG.md file, under the "Unreleased" tag.
+2. If necessary, update the package.json file with the release-it configuration
+   and the release script.
+3. Commit all changes.
+4. Run `yarn release –dry-run` to preview the release without making any actual
+   changes.
+5. Execute `yarn release` to create the release.
+
```

Comment:
could you add this part to the hardhat-example too ?

Reply:
done -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/1277/commits/52c0455869909df6e9837aee05a3c5ab8be611f6